rhv_checks.py

```diff
@@ -235,6 +236,63 @@ def check_storage_domain_attached_status(system, **kwargs):
         print("Ok: all Storage Domain(s) are Attached to Data Center(s): {}".format(okay))
         sys.exit(0)
 
+
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the hosts."""
+    okay, warning, critical, unknown, all = [], [], [], [], []
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
+        result = ssh_command(host.name, password, command)
+        host_data_list = []
+        host_data = {}
+        for data in result:
+            # if ' : ' in data:
+            if 'Host ID' in data or 'Engine status' in data or 'Hostname' in data:
+                key, value = data.split(' : ')
+                key = key.strip()
+                if key in host_data:
+                    host_data = {}
+                else:
+                    if host_data not in host_data_list:
+                        host_data_list.append(host_data)
+                    host_data[key] = value.strip(':')
+            host_data_list[0]['Engine status']
+
+        #TODO: Check status of each Engine Status and assert it.
```

Comment:
Please check:
1) when you run `hosted-engine --vm-status` command on each host, it should return status of all the hosts in the environment. 
2) Check Engine Status, Score, Health etc to assert OK state. Or at the very least you can assert 1 host is running engine and all others show it `Down` with `reason: VM not running on this host. ` And Ideal Score is supposed to be 3400 when Host is correctly running.

Reply:
Update logic.